docs/sources/reference/components/otelcol/otelcol.exporter.splunkhec.md

```diff
@@ -0,0 +1,210 @@
+
+#### otel_to_hec_fields block
+
+Name                       | Type     | Description                                                     | Default                       | Required
+---------------------------|----------|-----------------------------------------------------------------|-------------------------------|---------
+`severity_text`            | `string` | Maps severity text field to a specific HEC field.                |   `""`                        | no
+`severity_number`          | `string` | Maps severity number field to a specific HEC field.              |   `""`                        | no
+
+
+#### heartbeat block    
+
+Name                       | Type     | Description                                                     | Default                       | Required
+---------------------------|----------|-----------------------------------------------------------------|-------------------------------|---------
+`interval`                 | `time.Duration`    | Time interval for the heartbeat interval, in seconds.            |   `0s`                         | no
+`startup`                  | `bool`   | Send heartbeat events on exporter startup.                       |   `false`                        | no
+
+
+#### telemetry block
+
+Name                       | Type                  | Description                                                     | Default                       | Required
+---------------------------|-----------------------|-----------------------------------------------------------------|-------------------------------|---------
+`enabled`                  | `bool`                | Enable telemetry inside the exporter.                            |   `false`                     | no
+`override_metrics_names`   | `map(string)`   | Override metrics for internal metrics in the exporter.           |                | no
+
+#### batcher block
+
+Name                       | Type                  | Description                                                     | Default                       | Required
+---------------------------|-----------------------|-----------------------------------------------------------------|-------------------------------|---------
+`enabled`                  | `bool`                | Whether to not enqueue batches before sending to the consumerSender.                            |   `false`                     | no
+`flush_timeout`             | `time.Duration`       | The time after which a batch will be sent regardless of its size    | `200ms`  | no
+`min_size_items`   | `uint`   |  The number of items at which the batch is sent regardless of the timeout           |  `8192`              | no
+`max_size_items`   | `uint`   | Maximum number of batch items, if the batch exeeds this value, it will be broken up into smaller batches. Must be greater than or equal to min_size_items. Setting this value to zero disables the maximum size limit.          |  `0`           |  no
```

Comment:
```suggestion
`max_size_items`   | `uint`   | Maximum number of batch items, if the batch exceeds this value, it will be broken up into smaller batches. Must be greater than or equal to min_size_items. Setting this value to zero disables the maximum size limit.          |  `0`           |  no
```